src/non-profits.html

```diff
@@ -78,8 +78,39 @@ <h2 class="section-heading">Do you have any questions?</h2>
 <section>
   <h2 class="section-heading">Start your journey here</h2>
   <p>Please fill in the form below to see how we can help bring your project to life.
```

Comment:
Something doesn't seem right with this line. Think the copy needs improving

@krissy or @clawey you are wordsmiths - what do you think?